packages/ui/src/IconButton.tsx

```diff
@@ -133,7 +133,7 @@ const IconButtonComponent: FC<IconButtonProps> = ({
       {Boolean(loading) ? (
         <ActivityIndicator color={color} size="small" />
       ) : (
-        <FontAwesome6 brand="solid" color={color} name={iconName} size={16} />
+        <FontAwesome6 color={color} name={iconName} size={16} solid />
```

Comment:
there are two other places we're using `brand="solid"` - should those also be changed to just `solid` ?